build.sbt

```diff
@@ -72,80 +71,94 @@ def aecorModule(id: String, description: String): Project =
   Project(id, file(s"modules/$id"))
     .settings(moduleName := id, name := description)
 
-lazy val core = aecorModule("core", "Aecor Core")
+lazy val core = crossProject(JSPlatform, JVMPlatform)
+  .crossType(CrossType.Pure)
+  .in(file(s"modules/core"))
+  .settings(moduleName := "core", name := "Aecor Core")
   .settings(aecorSettings)
   .settings(coreSettings)
 
-lazy val boopickleWireProtocol =
-  aecorModule("boopickle-wire-protocol", "Aecor Boopickle Wire Protocol derivation")
-    .dependsOn(core)
-    .settings(aecorSettings)
-    .settings(boopickleWireProtocolSettings)
+lazy val coreJS = core.js
+lazy val coreJVM = core.jvm
+
+lazy val boopickleWireProtocol = crossProject(JSPlatform, JVMPlatform)
```

Comment:
sadly this `crossProject` thingy has to be directly assigned to a `val`. can't extract it into a method